src/components/card/index.js

```diff
@@ -0,0 +1,47 @@
+import { avatar } from "../avatar";
+import "./card.css";
+
+/**
+ * @typedef {Object} CardProps
+ * @property {string} CardProps.nome - some description here
+ * @property {string} CardProps.cargo - some description here
+ * @property {string} CardProps.img - some description here
+ * @property {boolean} CardProps.expanded - some description here
+ *
+ * @param {CardProps} props
+ */
+export function card({ nome, cargo, img, expanded }) {
+  if (expanded) {
```

Comment:
The only difference between expanded version or not is just square version.

![image](https://github.com/dcTeam23/face2face/assets/608731/8b4a2249-c0fa-45aa-b65f-44e93804cf0f)

Do you need to add some difference to apply a different style